src/test/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformerTest.java

```diff
@@ -2390,6 +2390,91 @@ public void givenSscs2Or5FormAndNoConfidentiality_thenCaseDataValueIsNull(FormTy
         assertNull(appellantConfidentialityRequired);
     }
 
+    @Test
+    @Parameters({"SSCS2", "SSCS5", "SSCS1", "SSCS1U", "SSCS1PE", "SSCS1PEU"})
+    public void notAValidFormFalse(String formType) {
+        assertFalse(transformer.notAValidFormType(formType));
+    }
+
+    @Test
+    @Parameters({"SSCS", "SSCS55", "SSCS11"})
+    public void notAValidFormTrue(String formType) {
+        assertTrue(transformer.notAValidFormType(formType));
+    }
+
+    @Test
+    public void givenNullForm_thenThrowError() {
+        pairs.put(IS_BENEFIT_TYPE_TAX_CREDIT, true);
+
+        pairs.put("person1_title", APPELLANT_TITLE);
+        pairs.put("person1_first_name", APPELLANT_FIRST_NAME);
+        pairs.put("person1_last_name", APPELLANT_LAST_NAME);
+        pairs.put("person1_address_line1", APPELLANT_ADDRESS_LINE1);
+        pairs.put("person1_address_line2", APPELLANT_ADDRESS_LINE2);
+        pairs.put("person1_address_line3", APPELLANT_ADDRESS_LINE3);
+        pairs.put("person1_address_line4", APPELLANT_ADDRESS_LINE4);
+        pairs.put("person1_postcode", APPELLANT_POSTCODE);
+        pairs.put("person1_email", APPELLANT_EMAIL);
+        pairs.put("person1_mobile", APPELLANT_MOBILE);
+        pairs.put("is_paying_parent", "true");
+
+        ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
```

Comment:
(Optional) You could new-line these builder methods to make the code more readable - same applies to tests below.